compiler/AST/AstToText.cpp

```diff
@@ -1320,6 +1330,21 @@ void AstToText::appendExpr(CallExpr* expr, bool printingType)
       mText += "nonnilable ";
       appendExpr(expr->get(1), printingType);
     }
+    else if (!expr->isPrimitive(PRIM_UNKNOWN))
+    {
+      mText += "__primitive(\"";
+      mText += expr->primitive->name;
+      mText += "\", \"";
```

Comment:
Most of the uses I see of `__primitive` do not use string literals for the arguments other than the name of the primitive being called, so I believe it would be incorrect to always surround the arguments with quotation marks.  If quotation marks are needed, I think the sub-expression probably should be the one handling it.

Reply:
I'm so naive! 😅 I didn't realize this until your comment. I have covered this usecase now. 
Also, `var a = foo("this is", a, "function", call);` is this a valid usecase? If yes, I see that the documentation generated for this usecase is `var a = foo(this is, a, function, call);` which seems to be wrong. Should I also add fix for this usecase? Thanks :)

Reply:
I think that'd be great!  But also, feel free to handle that in a separate PR :)

Reply:
Fixing this usecase is breaking few tests. I think I'd be better if I handle it in a separate PR. Thanks :)

Reply:
Oh, and would you mind filing an issue to track this other problem?  If I don't hear from you in a few days I'll do it myself, but I wanted to make sure it was a tracked action item in case life gets in the way of you completing it

Reply:
Sure I'll file an issue in few hours :)